src/main/java/shop/pet/track/rest/restController.java

```diff
@@ -0,0 +1,73 @@
+package shop.pet.track.rest;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.format.annotation.DateTimeFormat;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import shop.pet.track.entity.Owner;
+import shop.pet.track.entity.Pet;
+import shop.pet.track.service.PetTrackerService;
+
+import java.sql.Date;
+import java.time.LocalDate;
+import java.util.List;
+import java.util.Set;
+
+@RestController
+@RequestMapping("/api")
+public class restController {
+    private PetTrackerService petTrackerService;
+
+    @Autowired
+    public restController(PetTrackerService petTrackerService) {
+        this.petTrackerService = petTrackerService;
+    }
+
+    @PostMapping("/owner")
+    public Owner addOwner(@RequestBody Owner owner) {
```

Comment:
The return type of all controller method should be ResponseEntity<ClassName>

and response should be 
return new ResponseEntity<>(body, HttpStatus.OK);